### PR TITLE
Reworking contributors to not create people

### PR DESCRIPTION
**Pull request by jeremyf**
TL;DR
- We exposed a Contributor attribute in the UI
- This was transformed to the :creator in the datastream
- When we persisted the object any literal for a contributor would be
  used to create a Person object

To Run the Conversion from Command Line

$ rails console

> require 'curate/migration_services/runner'
> runner = Curate::MigrationServices::Runner.new(container_namespace: "::Curate::MigrationServices::MigrationContainers::RelatedContributor")
> runner.run

To Run via Capistrano, add the following to your Capfile, then look
at the available cap tasks

require 'curate/migration_services/capistrano'

Details

For some confounding reason, the form element for Contributor was
being persisted to the persisted :creator predicate and not a
:contributor predicate.

The assumption for this transformation is that any triple with the
:creator predicate and an object that is a PID can safely be mapped
from the :creator predicate to the :contributor predicate. The reason
being that we have not exposed a means of appending PIDs to :creator
except via the UI. And we have not exposed the :creator attribute
to the UI.

In the case of Image, we've only ever exposed the :contributor
attribute.

See also: 0d57a81dd2d62191f7e0267def5459f93d42c21b
See also: ndlib/planning#137

JIRA References:
https://jira.duraspace.org/browse/HYDRASIR-275

### CODE REVIEW

File: app/controllers/curation_concern/images_controller.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class CurationConcern::ImagesController < CurationConcern::GenericWorksController
   self.curation_concern_type = Image
+
+  def setup_forom
```
Comment (spr7b) [inline app/controllers/curation_concern/images_controller.rb:4]: Is 'forom' a typo ?  setup_form ??

File: app/views/application/_thumbnail.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<% width = 36 if width.blank? %>
+<%  if thumbnail.image? || thumbnail.pdf? || thumbnail.video? %>
+  <% path = download_path(thumbnail, {datastream_id: 'thumbnail'}) %>
+<% else %>
+  <% path = "curate/default.png" %>
+<% end %>
+<%= link_to image_tag(path, width: width, class: 'thumbnail'), curation_concern_generic_file_path(thumbnail.noid), class: 'canonical-image' %>
```
Comment (spr7b) [inline app/views/application/_thumbnail.html.erb:7]: Was this something left out of another HYDRASIR?  What does it have to do with contributors?

**Comment by mstroming** [comment]
Sue, good comments. I will merge because the code looks good and the tests pass. This will require a bunch of regression testing to make sure things work correctly.
